src/components/login-page/auth/utils.ts

```diff
@@ -18,10 +19,19 @@ export const fetchAndSetUser: () => Promise<void> = async () => {
       username: me.username,
       displayName: me.displayName,
       photo: me.photo,
-      provider: LoginProvider.LOCAL, // TODO Use real provider instead
+      loginProvider: AuthProviderType.LOCAL, // TODO Use real provider instead
       email: me.email
     })
   } catch (error) {
     console.error(error)
   }
 }
+
+/**
+ * Filter to apply to a list of auth providers to get only one-click providers.
+ * @param provider The provider to test whether it is a one-click provider or not.
+ * @return true when the provider is a one-click one, false otherwise.
+ */
+export const filterOneClickProviders = (provider: AuthProvider): boolean => {
+  return provider.type !== AuthProviderType.LOCAL && provider.type !== AuthProviderType.LDAP
```

Comment:
If you turn AuthProviderTypeOneClick (in types.ts) into an array you could just do an includes check.

Reply:
Sounds like a good idea.

Reply:
Done